app/models/alchemy/element.rb

```diff
@@ -57,7 +57,7 @@ class Element < ActiveRecord::Base
     scope :excluded,          ->(names) { where(arel_table[:name].not_in(names)) }
     scope :not_in_cell,       -> { where(cell_id: nil) }
     scope :in_cell,           -> { where("#{self.table_name}.cell_id IS NOT NULL") }
-    scope :from_current_site, -> { where(alchemy_languages: {site_id: Site.current || Site.default}).joins(page: 'language') }
+    scope :from_current_site, -> { where(Language.table_name => {site_id: Site.current || Site.default}).joins(page: 'language') }
```

Comment:
Line is too long. [130/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.